Public/Sdk/Public/Deployment/definition.dsc

```diff
@@ -53,7 +53,7 @@ export interface DeployedFileWithProvenance {
 @@public
 export interface FlattenedResult {
     flattenedFiles: Map<RelativePath, DeployedFileWithProvenance>,
-    flattenedOpaques: Map<RelativePath, OpaqueDirectory>,
+    flattenedOpaques: Map<RelativePath, [OpaqueDirectory, RelativePath]>,

```

Comment:
TODO: maybe add a comment explaining that `[OpaqueDirectory, RelativePath]` is a tuple of (1) an opaque directory, and (2) a path relative to that opaque directory designating a subdirectory to be added to the deployment